storybook/stories/API/props/utils.ts

```diff
@@ -5,6 +5,7 @@ export const getStoryArgsFromArgsTypesObject = (argsTypes: StorybookArgs): Recor
   const args: Record<string, unknown> = {};
   Object.keys(argsTypes).forEach((key: string) => {
     const defaultValue = argsTypes[key]?.defaultValue ?? argsTypes[key]?.table?.defaultValue;
+    // TODO this erases zeroes and `False` values, perhaps it should check for undefined instead?
     if (defaultValue) {
```

Comment:
Probably

Reply:
Couple of stories break when I do that. I'll fix it in separate PR